.github/workflows/ci.yml

```diff
@@ -30,670 +30,129 @@ jobs:
-          weblog: jersey-grizzly2
-        - library: java
-          weblog: ratpack
-        - library: java
-          weblog: resteasy-netty3
-        - library: java
-          weblog: vertx3
-        - library: java
-          weblog: spring-boot-jetty
-        - library: java
-          weblog: spring-boot
-        - library: java
-          weblog: uds-spring-boot
-        - library: java
-          weblog: spring-boot-openliberty
-        - library: java
-          weblog: spring-boot-undertow
-        - library: nodejs
-          weblog: express4
-        - library: nodejs
-          weblog: uds-express4
-        - library: nodejs
-          weblog: express4-typescript
-        - library: php
-          weblog: apache-mod-7.0
-        - library: php
-          weblog: apache-mod-7.1
-        - library: php
-          weblog: apache-mod-7.2
-        - library: php
-          weblog: apache-mod-7.3
-        - library: php
-          weblog: apache-mod-7.4
-        - library: php
-          weblog: apache-mod-8.0
-        - library: php
-          weblog: apache-mod-8.1
-        - library: php
-          weblog: apache-mod-7.0-zts
-        - library: php
-          weblog: apache-mod-7.1-zts
-        - library: php
-          weblog: apache-mod-7.2-zts
-        - library: php
-          weblog: apache-mod-7.3-zts
-        - library: php
-          weblog: apache-mod-7.4-zts
-        - library: php
-          weblog: apache-mod-8.0-zts
-        - library: php
-          weblog: apache-mod-8.1-zts
-        - library: php
-          weblog: php-fpm-7.0
-        - library: php
-          weblog: php-fpm-7.1
-        - library: php
-          weblog: php-fpm-7.2
-        - library: php
-          weblog: php-fpm-7.3
-        - library: php
-          weblog: php-fpm-7.4
-        - library: php
-          weblog: php-fpm-8.0
-        - library: php
-          weblog: php-fpm-8.1
-        - library: python
-          weblog: flask-poc
-        - library: python
-          weblog: django-poc
-        - library: python
-          weblog: uwsgi-poc
-        - library: python
-          weblog: uds-flask
-        - library: ruby
-          weblog: rack
-        - library: ruby
-          weblog: sinatra14
-        - library: ruby
-          weblog: sinatra20
-        - library: ruby
-          weblog: sinatra21
-        - library: ruby
-          weblog: uds-sinatra
-        - library: ruby
-          weblog: rails32
-        - library: ruby
-          weblog: rails40
-        - library: ruby
-          weblog: rails41
-        - library: ruby
-          weblog: rails42
-        - library: ruby
-          weblog: rails50
-        - library: ruby
-          weblog: rails51
-        - library: ruby
-          weblog: rails52
-        - library: ruby
-          weblog: rails60
-        - library: ruby
-          weblog: rails61
-        - library: ruby
-          weblog: rails70
-        version:
-        - prod
-        - dev
+      #max-parallel: 2
```

Comment:
coud we use max-parallel instead of having tree jobs? (test-the-test-0, test-the-test-1, test-the-test-2)